ginga/misc/plugins/Cuts.py

```diff
@@ -120,6 +122,7 @@ def __init__(self, fv, fitsimage):
                                   show_cuts_legend=False, enable_slit=False)
         self.settings.load(onError='silent')
         self.colors = self.settings.get('colors', cut_colors)
+        self.show_legend = self.settings.get('show_cuts_legend', False)
```

Comment:
In this case it's fine.  Just wanted you to realize the tradeoff for future updates to plugins.  If there is any idea that the plugin should respond without restarting/reloading to changes in preferences it is preferable to keep the state confined to the settings object.